src/codemagic/tools/app_store_connect/actions/fetch_signing_files_action.py

```diff
@@ -0,0 +1,237 @@
+from __future__ import annotations
+
+from abc import ABCMeta
+from itertools import chain
+from typing import Iterator
+from typing import List
+from typing import Optional
+from typing import Sequence
+from typing import Tuple
+from typing import Union
+
+from codemagic import cli
+from codemagic.apple import AppStoreConnectApiError
+from codemagic.apple.resources import BundleId
+from codemagic.apple.resources import BundleIdPlatform
+from codemagic.apple.resources import CertificateType
+from codemagic.apple.resources import DeviceStatus
+from codemagic.apple.resources import Profile
+from codemagic.apple.resources import ProfileState
+from codemagic.apple.resources import ProfileType
+from codemagic.apple.resources import SigningCertificate
+from codemagic.cli import Colors
+from codemagic.models import PrivateKey
+
+from ..abstract_base_action import AbstractBaseAction
+from ..arguments import BundleIdArgument
+from ..arguments import CertificateArgument
+from ..arguments import CommonArgument
+from ..arguments import ProfileArgument
+from ..arguments import Types
+from ..errors import AppStoreConnectError
+
+
+class FetchSigningFilesAction(AbstractBaseAction, metaclass=ABCMeta):
+    @cli.action(
+        "fetch-signing-files",
+        BundleIdArgument.BUNDLE_ID_IDENTIFIER,
+        BundleIdArgument.PLATFORM,
+        CertificateArgument.PRIVATE_KEY,
+        CertificateArgument.PRIVATE_KEY_PASSWORD,
+        CertificateArgument.P12_CONTAINER_PASSWORD,
+        ProfileArgument.PROFILE_TYPE,
+        BundleIdArgument.IDENTIFIER_STRICT_MATCH,
+        CommonArgument.CREATE_RESOURCE,
+    )
+    def fetch_signing_files(
```

Comment:
Used to be defined in https://github.com/codemagic-ci-cd/cli-tools/blob/3670fa624997f17cb437bc88198f7bdee3594f35/src/codemagic/tools/app_store_connect.py#L825-L883